src/Cache/Cache.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Cache;
+
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use InvalidArgumentException;
+
+class Cache implements CacheInterface
+{
+    /** @var CacheItemPoolInterface */
+    private $cachePool;
+
+    /** @var string */
+    private $prefix;
+
+    /** @var bool */
+    private $flushCacheItems = false;
+
+    private $defaultCacheTimes = [
+        CacheInterface::NONE => -1,
```

Comment:
If you want to store something in the cache and have it instantly expire then -1. If you want to store something indefinitely then 0